spec/GameSpec.js

```diff
@@ -0,0 +1,131 @@
+const Game = require("../src/Game.js");
```

Comment:
These are unit tests for Game, but they are not isolated from Frame. If Frame was broken, all of these would fail too. Unit tests should just test the specific class. Think about what Game needs to do for each method, and test its behaviour. You can pass in a double Frame class to isolate from Frame, then test what methods Game sends to the pretend frames.